cert/vault_source.go

```diff
@@ -119,15 +124,21 @@ func (s *VaultSource) checkRenewal(c *api.Client) {
 		return
 	}
 
-	if data.Renewable {
-		s.auth.renewIncrement = data.CreationTTL
+	switch {
+	case data.Renewable:
+		s.auth.renewTTL = data.CreationTTL
 		s.auth.expireTime = data.ExpireTime
-	} else if !data.ExpireTime.IsZero() && !dropNotRenewableWarning {
-		ttl := time.Until(data.ExpireTime)
-		ttl = ttl / time.Second * time.Second // truncate to seconds
-		log.Printf("[WARN] vault: Token is not renewable and will expire %s from now (at %s)",
-			ttl, data.ExpireTime.Format(time.RFC3339))
+	case data.ExpireTime.IsZero():
+		// token doesn't expire
+		return
+	case dropNotRenewableWarning:
+		return
 	}
+
+	ttl := time.Until(data.ExpireTime)
+	ttl = ttl / time.Second * time.Second // truncate to seconds
+	log.Printf("[WARN] vault: Token is not renewable and will expire %s from now (at %s)",
```

Comment:
The logging is the `default` case, right?

pls change the log msg to `Token is not renewable and expires in %s from now at %s`